samples/core/resource_ops/resourceop_basic.py

```diff
@@ -18,7 +18,7 @@
 It is not a good practice to put password as a pipeline argument, since it will
 be visible on KFP UI.
 """
-
+import kfp
```

Comment:
nit: can you switch the order of following two imports. thx.

Reply:
NP